frontend/src/v5/ui/components/shared/latestRevision/latestRevision.component.tsx

```diff
@@ -17,34 +17,27 @@
 
 import { FormattedMessage } from 'react-intl';
 
-import { RevisionStatus } from '@/v5/ui/routes/dashboard/projects/containers/containersList/latestRevision/revisionStatus';
-import { IRevisionStatus } from '@/v5/ui/routes/dashboard/projects/containers/containersList/latestRevision/revisionStatus/revisionStatus.component';
-import { UploadStatus } from '@/v5/store/containers/containers.types';
+import { ContainerUploadStatus } from '@/v5/store/containers/containers.types';
 import { Container, Label } from './latestRevision.styles';
+import { IRevisionStatus, RevisionStatus } from './revisionStatus/revisionStatus.component';
 
 interface ILatestRevision extends IRevisionStatus {
 	hasRevisions: boolean;
+	emptyLabel: string;
 }
 
-export const LatestRevision = ({ hasRevisions, status, ...props }: ILatestRevision): JSX.Element => (
+export const LatestRevision = ({ hasRevisions, status, emptyLabel, ...props }: ILatestRevision): JSX.Element => (
 	<Container disabled={!hasRevisions}>
-		{hasRevisions || status !== UploadStatus.OK ? (
+		{hasRevisions || status !== ContainerUploadStatus.OK ? (
```

Comment:
I was thinking about the separation introduced for UploadStatus with "DrawingUploadStatus" and "ContainerUploadStatus". As there as some components shared between drawings and containers, they shouldn't have any specific knowledge about either, but some, like this one, do. For instance, here you checking the value of the status against `ContainerUploadStatus` which is no longer the same as for drawings